opencart 1.5.6.4/upload/catalog/model/smailyforopencart/helper.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+class ModelSmailyForOpencartHelper extends Model {
+    /**
+     * Makes cUrl call to smaily api endpoint. Returns body or empty if error.
```

Comment:
`cUrl` should be written as `cURL`, and `api` as `API`. When in doubt, Google it :)